plugins/ern_v0.48.0+/@shopify/flash-list_v1.0.4+/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "android": {
+    "root": "",
+    "moduleName": "android",
+    "modulePath": "src/main/kotlin",
+    "dependencies": [
+      "androidx.appcompat:appcompat:1.2.0",
+      "androidx.core:core-ktx:1.6.0"
+    ],
+    "copy": [{ "source": "android/", "dest": "lib/src/main/assets/fonts" }]
```

Comment:
Shouldn't the `source` be more specific? or the only things inside the android folder are the fonts? 🤔

Reply:
ohh then can't the "copy" property be removed?

Reply:
It was actually not necessary. I removed it [a71b367](https://github.com/PicnicSupermarket/electrode-native-manifest/pull/11/commits/a71b367ead27cdecb45a566482f7db28358a17f5) here.